src/ui/public/index_patterns/__tests__/_index_pattern.test.js

```diff
@@ -63,17 +63,21 @@ jest.mock('../_intervals', () => ({
   IndexPatternsIntervalsProvider: jest.fn(),
 }));
 
-jest.mock('../_field_list', () => ({
-  IndexPatternsFieldListProvider: jest.fn().mockImplementation((pattern) => {
-    return {
-      byName: {
-        id: { value: pattern.id },
-        title: { value: pattern.title },
-      },
-      every: jest.fn(),
-    };
-  })
-}));
+jest.mock('ui/chrome',
```

Comment:
Why was this added? I tried running the tests without it and there didn't seem to be any difference.